cardano-crypto-class/src/Cardano/Crypto/Seed.hs

```diff
@@ -58,6 +59,18 @@ getBytesFromSeed n (Seed s)
   where
     (b, s') = BS.splitAt n s
 
+-- | Split a seed into two smaller seeds, the first of which is the given
+-- number of bytes large, and the second is the remaining. This will fail if
+-- not enough bytes are available. This can be chained multiple times provided
+-- the seed is big enough to cover each use.
+--
+splitSeed :: Int -> Seed -> Maybe (Seed, Seed)
+splitSeed n (Seed s)
```

Comment:
Repetition of `getBytesFromSeed`, but since it is so small, it's no big deal. 

Alternatively:
```haskell
splitSeed n seed = first Seed <$> getBytesFromSeed n seed
```